harness/determined/launch/horovod.py

```diff
@@ -19,7 +19,9 @@
 from determined.constants import DTRAIN_SSH_PORT
 
 
-def create_sshd_worker_cmd(allocation_id: str, num_slot_ids: int) -> Tuple[List[str], List[str]]:
+def create_sshd_worker_cmd(
```

Comment:
@rb-determined-ai can you just look at this file instead of my pr. @jerryharrow wrote it so I'll let him pr it.